addons/stock_account/models/stock_valuation_layer.py

```diff
@@ -146,13 +146,15 @@ def _consume_all(self, qty_valued, valued, qty_to_value):
         if not self:
             return 0, 0
 
-        rounding = self.product_id.uom_id.rounding
+        min_rounding = 1.0
         qty_total = -qty_valued
         value_total = -valued
         new_valued_qty = 0
         new_valuation = 0
 
         for svl in self:
+            rounding = svl.product_id.uom_id.rounding
+            min_rounding = min(min_rounding, rounding)
```

Comment:
I don't understand this. This `min(1.0, something_positive)` equals to `1.0` always :S

Reply:
`rounding` can be a value in this format:
- 1.0
- 0.1
- 0.01
- 0.001
- 0.00000001

So `min` should return the most precise rounding from among the involved records IIUC.

Reply:
Ah, my bad, I thought it was the other rounding, the one stating the number of decimals you want to round up.